<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongMetadataSettingsPage</class>
 <widget class="QWidget" name="SongMetadataSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>1395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Song Metadata</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_max_tagclients">
     <property name="title">
      <string>Maximum number of child processes for tag handling (requires restart)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="numprocs_tagclients_label">
        <property name="text">
         <string>Number of processes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="max_numprocs_tagclients_value_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="max_numprocs_tagclients">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="guess_metadata">
     <property name="toolTip">
      <string>Try to extract title, artist, and album from file path.</string>
     </property>
     <property name="text">
      <string>Try to guess missing metadata</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
